code/code/misc/spell_parser.cc

```diff
@@ -1673,7 +1673,7 @@ namespace {
     { AFFECT_VASCO_BLESSING, "AFFECT_VASCO_BLESSING" },
     { AFFECT_CORAL_BLESSING, "AFFECT_CORAL_BLESSING" },
     { AFFECT_BUMP_BLESSING, "AFFECT_BUMP_BLESSING" },
-    { AFFECT_MAROR_BLESSING, "AFFECT_MAROR_BLESSING" },
+    { AFFECT_ONSLAUGHT_BLESSING, "AFFECT_ONSLAUGHT_BLESSING" },
```

Comment:
Why did you feel a need to kick Maror out?

Reply:
Was following Warluks suggestion to replace a current blessing for simplicity.  Maror was a random choice.

Reply:
Eh, why not. I guess it doesn't matter much.